static-tests: test error

#### Description

Error in the `static-tests` job, looks coccinelle can't be installed for `Jammy `
```sh
Err:8 https://ppa.launchpadcontent.net/npalix/coccinelle/ubuntu jammy Release
  404  Not Found [IP: 185.125.190.52 443]
Reading package lists...
E: The repository 'https://ppa.launchpadcontent.net/npalix/coccinelle/ubuntu jammy Release' does not have a Release file.
Error: Process completed with exit code 100.
```

#### Steps to reproduce the issue
The error is appears in each job, look for example https://github.com/Mesh4all/m4a-firmware/actions/runs/4449498332/jobs/7813766058?pr=375
![image](https://user-images.githubusercontent.com/10358374/226317922-3f2b6299-ea0e-4224-ac14-677cc1f1f0c6.png)


#### Expected results
No error

#### Actual results

#### Versions
This error occurs in the latest version of Ubuntu LTS, but not in the previous versions as Ubuntu 20.04 LTS

--- a/.github/workflows/static-test.yml
+++ b/.github/workflows/static-test.yml
@@ -19,7 +19,6 @@ jobs:
 
       - name: Setup code tools
         run: |
-          sudo add-apt-repository ppa:npalix/coccinelle
           sudo apt update
           sudo apt install codespell cppcheck vera++ flake8 uncrustify pcregrep shellcheck doxygen coccinelle
 

